messages/message_handler.go

```diff
@@ -27,7 +27,7 @@ type MessageHandler interface {
 
 	// SendMessage sends the signed message to the destination chain. The payload parsed according to
 	// the VM rules is also passed in, since MessageManager does not assume any particular VM
```

Comment:
Can you add to the interface comment what the returned common.Hash is intended to represent?